aviary/validation_cases/benchmark_tests/test_bench_FwFm.py

```diff
@@ -286,7 +286,7 @@ def setUp(self):
                     "initial_bounds": ((0.0, 2.0), "min"),
                     "duration_bounds": ((5.0, 50.0), "min"),
                     "no_descent": False,
-                    "add_initial_mass_constraint": False,
+                    "add_initial_mass_constraint": True,
                 },
```

Comment:
Ultimately, we are going to have to add more tests/benchmarks for all of these options to have complete coverage. We might not need full optimizations, but we do need to build and inspect.  Same is true with optimize_mass/altitude, etc.  Because of interactions, we might need a full-factorial number of tests.

Reply:
Agreed! Made an issue: https://github.com/OpenMDAO/om-Aviary/issues/108